#request fields
float64 x
float64 y
float64 z
uint8 transfer_state
---
uint8 transfer_state